### DISCUSSION TITLE
Change email address in github.community

### DISCUSSION THREAD

**Discussion by HoodFeelGood** [community/community#23229] [New to GitHub]
<p>I changed my primary email address on Github and removed the previous email address. I went into Github.community and unlinked and relinked my Github account.</p>
<p>Github.community correctly shows my Github account with the new email address as the “Associated Account” however the Github.community primary email address is still the old one. I can’t find a way to change it… please help if you can.</p>
<p>Here is a screenshot<br>
<a href="/uploads/short-url/7ubsAO1vgVkJOOmc5P3KDiLJEJr.png">Screenshot 2021-01-02 163654|276x499</a></p>

**Comment by canuckjacq** [comment]
<p>This is a weird one. Can you confirm that you’re receiving GitHub community emails to the old email address?</p>

**Comment by HoodFeelGood** [comment]
<p>Yes. Just got one indicating your response to this thread.</p>

**Comment by canuckjacq** [comment]
<p>Thanks for clarifying.</p>
<p>I’ve just tested this and managed to change my email by disconnecting and then reconnecting my GitHub account in my preferences:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/4/740af903a94421d5d19a0e24c11b253f0913b93a.png" data-download-href="/uploads/short-url/gyyTwizEQ3vNnovPkdaaOU5HHcK.png?dl=1" title="Screenshot 2021-01-04 at 16.36.13"><img src="https://user-images.githubusercontent.com/11161530/181075670-d635a8e3-0f25-4ef6-aeaa-245c149f4784.png" alt="Screenshot 2021-01-04 at 16.36.13" data-base62-sha1="gyyTwizEQ3vNnovPkdaaOU5HHcK" width="345" height="91" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/4/740af903a94421d5d19a0e24c11b253f0913b93a_2_345x91.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/4/740af903a94421d5d19a0e24c11b253f0913b93a_2_517x136.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/4/740af903a94421d5d19a0e24c11b253f0913b93a_2_690x182.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/4/740af903a94421d5d19a0e24c11b253f0913b93a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2021-01-04 at 16.36.13</span><span class="informations">700×186 8.53 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Can you try that?</p>

**Comment by HoodFeelGood** [comment, marked as answer]
<p>I just tried again. I had done that the first time. This time, after disconnecting account I hit save. Then I reconnected account and hit save. Then I logged out. Then I logged back in. Then it worked!</p>
<p>Thanks!</p>
